mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -7961,6 +7991,22 @@
 															"    pm.expect(tv4.missing.length).to.equal(0, \"Missing schemas: \" + JSON.stringify(tv4.missing));",
 															"});",
 															"",
+															"//Check that tagList is not empty",
+															"pm.test('Creating tagList', function(){",
+															"    pm.expect(response.data.attributes.tags.tagList.size) !== 0;",
+															"});",
+															"    ",
+															"//Check that tagList is not empty",
+															"pm.test('Creating tagList', function(){",
+															"    pm.expect(response.data.attributes.tags.tagList[0]).eq(\"foo-tag\");",
+															"    pm.expect(response.data.attributes.tags.tagList[1]).eq(\"another-tag\");",
+															"});",
+															"",
+															"//Check that tagList has 2 elements",
+															"pm.test('Creating tagList', function(){",
+															"    pm.expect(response.data.attributes.tags.tagList.size) === 2;",
+															"});",
+															"",
```

Comment:
Same comment as above on this check happening after attributes check

Reply:
Thanks @carolegodfrey . All issues have been fixed.